profile/README.md

```diff
@@ -10,8 +10,8 @@ To work as a community to facilitate adoption of IETF Supply Chain Integrity, Tr
 + Join the SCITT community mailing list at https://groups.io/g/scitt-community
 
 ## SCITT Community Meetings:
-+ SCITT Community Meetings: Bi-Weekly on Monday 08:00 PDT/11:00 EDT/15:00 UTC
-+ SCITT Community Bi-Weekly Meeting (iCal link): [Click Here](https://meet.google.com/rek-mbak-nxv)
++ SCITT Community Meetings: Bi-Weekly on Monday [08:00 PDT/11:00 EDT/15:00 UTC](https://time.is/compare/300PM_5_June_2024_in_UTC/Tokyo/California/New_York/London)
```

Comment:
The time.is is a cool view. Could we add the iCal link here as well? Or, a link to the community page so we have one place to update if/when the meeting changes?

Reply:
Argh, that's what I meant.  How did I screw that up? 😂